core/arch/arm/mm/tee_mmu.c

```diff
@@ -496,245 +496,240 @@ static TEE_Result param_mem_to_user_va(struct user_ta_ctx *utc,
 	return TEE_ERROR_GENERIC;
 }
 
+static int cmp_param_mem(const void *a0, const void *a1)
+{
+	const struct param_mem *m1 = a1;
+	const struct param_mem *m0 = a0;
+	int ret;
+
+	/* Make sure that invliad param_mem are placed last in the array */
+	if (m0->size == 0 && m1->size == 0)
```

Comment:
minor: not going the `if (!m0->size)` way :)?

Reply:
Must have been a Freudian slip, I'll change.